src/server.js

```diff
@@ -53,10 +54,22 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
-		server = server.createServer({
-			key: fs.readFileSync(keyPath),
-			cert: fs.readFileSync(certPath)
-		}, app);
+		if (config.https.ca.length && fs.existsSync(caPath)) {
```

Comment:
This logic doesn't seem right.

I'm assuming you mean `(config.https.ca.length && !fs.existsSync(caPath))`

Also, you'd probably want to do "caPath" rather than "config.https.ca", yeah?